benchmark.py

```diff
@@ -0,0 +1,102 @@
+# -*- coding: utf-8 -*-
+
+from __future__ import absolute_import, unicode_literals
+
+import cProfile
+import logging
+import os
+from pstats import Stats
+from memory_profiler import profile
+
+from draftjs_exporter.constants import BLOCK_TYPES, ENTITY_TYPES
+from draftjs_exporter.defaults import BLOCK_MAP, STYLE_MAP
+from draftjs_exporter.dom import DOM
+from draftjs_exporter.html import HTML
+
+from markov_draftjs import get_content_sample
```

Comment:
This content sample is about 2.3Mb of text, which is why it's pulled from a separate package.